master/security/protect-hosts.md

```diff
@@ -0,0 +1,182 @@
+|   6666 |   TCP    |  Inbound & Outbound |             etcd self-hosted service access    |
+|   6667 |   TCP    |  Inbound & Outbound |             etcd self-hosted service access    |
+
+#### Use policy to restrict host traffic 
+
+##### Step 1: Create policy to restrict host traffic
+
+Although failsafe rules provide protection from removing all connectivity to a host, you should create a global network policy policy that restrict host traffic. 
+
+In the following example, we use a **GlobalNetworkPolicy** that applies to all worker nodes (defined by a label). Ingress SSH access is allowed from a defined "management" subnet. 
+
+**Ingress traffic** is also allowed for ICMP, and on TCP port 10250 (default kubelet port). **Egress** traffic is allowed to etcd on a particular IP, and UDP on port 53 and 67 for DNS and DHCP.
+
+```
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: k8s-worker
+spec:
+  selector: "role == 'k8s-worker'"
+  order: 0
+  ingress:
+  - action: Allow
+    protocol: TCP
+    source:
+      nets:
+      - "<your management CIDR>"
+    destination:
+      ports: [22]
+  - action: Allow
+    protocol: ICMP
+  - action: Allow
+    protocol: TCP
+    destination:
+      ports: [10250]
+  egress:
+  - action: Allow
+    protocol: TCP
+    destination:
+      nets:
+      - "<your etcd IP>/32"
+      ports: [2379]
+  - action: Allow
+    protocol: UDP
+    destination:
+      ports: [53, 67]
+ ```
+
+##### Step 2: Create host endpoints
+
+For each host point that you want to secure with policy, you must create a **host endpoint object**. To do that, you need the name of the Calico node on the host that owns the interface; in most cases, it is the same as the hostname of the host. 
+
+In the following example, we create a host endpoint for the host named **my-host** with the interface named **eth0**, with **IP 10.0.0.1**. Note that the value for **node:** must match the hostname used on the Calico node object. 
+
+When the host endpoint is created, traffic to or from the interface is dropped unless policy in place. 
+
+```
+apiVersion: projectcalico.org/v3
+kind: HostEndpoint
+metadata:
+  name: my-host-eth0
+  labels:
+    role: k8s-worker
+    environment: production
+spec:
+  interfaceName: eth0
+  node: my-host
+  expectedIPs: ["10.0.0.1"]
+```
+#### Control default behavior of workload endpoint to host traffic  
+
+The default Calico behavior blocks all connections from workloads to their local host (after traffic passes any egress policy applied to the workload). You can change this behavior using the **DefaultEndpointToHostAction** parameter in Felix configuration. 
+
+This parameter works at the IP table level, where you can specify packet behavior to **Drop** (default), **Accept**, or **Return** (if you have your own rules in IP tables). 
+
+To change this parameter for all hosts, edit the **FelixConfiguration** object named “default.”
+
+1. Get a copy of the object to edit.  
```

Comment:
The formatting here is a bit off. I think it should look like this (replacing single-quotes with backticks)

1. Get a copy of the object to edit

    '''
    calicoctl get felixconfiguration default --export -o yaml > default-felix-config.yaml
    '''

Reply:
fixed.